ubuntu/python/Dockerfile

```diff
@@ -23,11 +23,14 @@ RUN virtualenv --python=python${python_version} --system-site-packages /databric
 
 # These python libraries are used by Databricks notebooks and the Python REPL
 # You do not need to install pyspark - it is injected when the cluster is launched
-# Versions are intended to reflect latest DBR: https://docs.databricks.com/release-notes/runtime/13.3.html#system-environment
+# Versions are intended to reflect latest DBR LTS: https://docs.databricks.com/en/release-notes/runtime/15.4lts.html#system-environment 
 
 COPY requirements.txt /databricks/.
 
-RUN /databricks/python3/bin/pip install -r /databricks/requirements.txt
+RUN apt-get install -y libpq-dev build-essential 
```

Comment:
we need build-essential?

Reply:
yes, need c++ compiler to compile some python dependencies